eth_crowdfund_be/eth_crowdfund_api/src/seeds/seed.py

```diff
@@ -0,0 +1,110 @@
+from ..models import db
+from ..models.Campaign import Campaign
+from ..models.Request import Request
+import click
+
+@click.command()
+@click.option('--seed', default=False, help='Seed database?')
+def add_seeds(seed):
+    if seed == False: 
+        return 
+    
+    click.echo('Seeding...')
+
+#  Campaign Seeds
+    campaign1 = {
+        'name': 'Build a Well', 
+        'description': 'A small town in South Africa needs a well for clean water', 
+        # Uses lorem picsum for random pics
+        'image': 'https://picsum.photos/200/300', 
+        # Manager is a random string
+        'manager': 'X5JT498FJeklnsd8382hf',
```

Comment:
Currently random. We may need to update these to actually point somewhere

Reply:
I agree. I think we can have these point to an actual manager on the test network once we have those created.